DigitalLearningSolutions.Data/Helpers/ConfigHelper.cs

```diff
@@ -41,6 +42,11 @@ public static bool IsSignpostingUsed(this IConfiguration config)
             return bool.Parse(config[UseSignposting]);
         }
 
+        public static bool IsPricingPageUsed(this IConfiguration config)
+        {
+            return bool.Parse(config[PricingPage]);
+        }

```

Comment:
Could just tweak the names here to make it slightly nicer - `IsPricingPageUsed` -> `IsPricingPageEnabled` and `PricingPage` -> `(Is)PricingPageEnabled`. It's not quite appropriate to follow the same pattern as signposting because signposting is a feature / process / concept where as PricingPage is a page.
But other than that looks great!

Reply:
Yep, changed the names :)

Reply:
Ah, sorry, I just meant the c# constant `PricingPage` rather than the config setting key, but I now realise my message was a bit unclear on that part. I think the config setting is better being called `PricingPage`, because that fits in with the convention we're using. Could you change it back and then just go ahead and merge? No need for me to re-review.

Reply:
Yep, changed the key back